package.json

```diff
@@ -28,16 +28,25 @@
         "@babel/core": "7.4.0",
         "@babel/plugin-transform-runtime": "7.4.0",
         "@babel/preset-env": "7.4.1",
+        "@babel/preset-typescript": "^7.12.1",
```

Comment:
I would prefer to move away from babel if we can build with `tsc`.

Reply:
ok, let me try to remove it